release_notes/RELEASE.txt

```diff
@@ -180,6 +180,23 @@ New features and changes
       * bitrec_t struct
       * maskc and maskl variable declarations
 
+    - Most of hfile.h moved to hfile_priv.h
+
+      Still in hfile.h:
+
+      LIBVER_MAJOR
+      LIBVER_MINOR
+      LIBVER_RELEASE
+      LIBVER_SUBRELEASE
+      LIBVER_STRING
+      LIBVSTR_LEN
+      LIBVER_LEN
+
+      Everything else has been moved to hfile_priv.h
+
+      NOTE that hfile.h is NOT included via hdf.h and must be included
```

Comment:
Should the behavior be changed? Would it be something users would always need to do?

Reply:
I thought about it, but I don't like adding the generic name to the namespace. If it were H4_<whatever>, I would have done that, but not with something that could cause conflicts.